web/src/less/preview-image-carousel-dialog.less

```diff
@@ -0,0 +1,34 @@
+@import "./mixin.less";
+
+.preview-image-dialog {
```

Comment:
It's the same classname with `PreviewImageDialog.tsx`, rename it to `preview-image-carousel-dialog`?